docs/application/toc_all.md

```diff
@@ -305,7 +312,15 @@
 
 ## [Overview](/application/web/index.md)
 
-## Get Started
+## Get Started using VSCode
```

Comment:
@dkson95 In current order, VS Code extension guide comes before than Visual Studio guide.
I'd like to know if it matches with your intend.

Same issue  to native guides.

Reply:
> @dkson95 In current order, VS Code extension guide comes before than Visual Studio guide. I'd like to know if it matches with your intend.
> 
> Same issue to native guides.

Right. Agreed on this :)